### PR TITLE
Add links to UG commands

### PR DESCRIPTION
**Pull request by axmszr**

### CODE REVIEW

**Review by markusyeo — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -366,9 +366,9 @@ Format: `list`
 
 As `list` resets all active filters, one of the most common uses of the `list` command to reset the display lists to display all the cases in `DengueHotspotTracker`.
 You might like to use `list` in conjunction with the following commands:
-* [`delete`](#deleting-cases-delete) to show all the cases as delete follows the index of the current list
+* [`delete`](#deleting-cases--delete) to show all the cases as delete follows the index of the current list
```
Comment (axmszr) [inline docs/UserGuide.md:369]:
```suggestion
* [`delete`](#deleting-cases-delete) to show all the cases as delete follows the index of the current list
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -366,9 +366,9 @@ Format: `list`
...
 * [`overview`](#changing-the-overview-type--overview), which shows an overview of the cases in the current list view
-* [`find`](#finding-cases-by-prefixes-find) which filters the current list view
+* [`find`](#finding-cases-by-prefixes--find) which filters the current list view
```
Comment (axmszr) [inline docs/UserGuide.md:371]:
```suggestion
* [`find`](#finding-cases-by-prefixes-find) which filters the current list view
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -366,9 +366,9 @@ Format: `list`
 
 As `list` resets all active filters, one of the most common uses of the `list` command to reset the display lists to display all the cases in `DengueHotspotTracker`.
 You might like to use `list` in conjunction with the following commands:
-* [`delete`](#deleting-cases-delete) to show all the cases as delete follows the index of the current list
+* [`delete`](#deleting-cases--delete) to show all the cases as delete follows the index of the current list
 * [`overview`](#changing-the-overview-type--overview), which shows an overview of the cases in the current list view
```
Comment (axmszr) [inline docs/UserGuide.md:370]:
```suggestion
* [`overview`](#changing-the-overview-type-overview), which shows an overview of the cases in the current list view
```

**Review by axmszr — COMMENTED** [review]
Some links to fix

**Pull request merged**